taca/log/__init__.py

```diff
@@ -19,16 +20,16 @@
 }
 
 def init_logger_file(log_file, log_level='INFO'):
-    """ Append a FileHandler to the general logger.
+    """ Append a FileHandler to the root logger.
 
     :param str log_file: Path to the log file
     :param str log_level: Logging level
     """
     log_level = LOG_LEVELS[log_level] if log_level in LOG_LEVELS.keys() else logging.INFO
 
-    LOG.setLevel(log_level)
+    ROOT_LOG.setLevel(log_level)
```

Comment:
And isn't that a good behaviour :)? If you run TACA interactively you'll get the logs in the console (and in the file), and if you run TACA as a cronjob you'll still get the logs in the file.